generate.go

```diff
@@ -1,4 +1,38 @@
-//go:generate go-bindata -pkg=data -ignore=.*\.go -o data.go . ./benkeen/d3pie/ ./jiahuang/d3-timeline/ ./krisk/fuse/ ./zeroclipboard/zeroclipboard/ ./seiyria/bootstrap-slider/
-//go:generate go fmt .
+// +build generate
 
-package data
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+	pathpkg "path"
+
+	"github.com/shurcooL/go/vfs/httpfs/filter"
+	"github.com/shurcooL/vfsgen"
+)
+
+func main() {
+	ignore := func(fi os.FileInfo, path string) bool {
+		switch {
+		case len(fi.Name()) >= 2 && fi.Name()[0] == '.': // Skip .files/.folders (but not ".").
+			return true
+		case pathpkg.Ext(fi.Name()) == ".go":
+			return true
+		case path == "/README.md":
+			return true
+		default:
+			return false
+		}
+	}
+	var data http.FileSystem = filter.NewIgnore(http.Dir("."), ignore)
+
+	err := vfsgen.Generate(data, vfsgen.Options{
+		Filename:     "data.go",
+		PackageName:  "data",
+		VariableName: "Assets",
+	})
+	if err != nil {
+		log.Fatalln(err)
+	}
+}
```

Comment:
> I would argue there is _no reason to not have it exported by default_, given that nobody will import main packages anyway so that only leaves library packages. But if you disagree then it's A-OK to keep as-is.

Good point. The reason I opted to have unexported by default is because my general policy about naming Go variables is to always use unexported as a default, and upgrade to exported if it **needs** to be. Even inside Go commands (binaries), I always start out with all unexported global identifiers, and upgrade them to exported only when there's a need (usually there isn't).

In this case, as you point out, it seems that while there's no need for it for commands, when using `vfsgen` for a library, exported is a better default.

Since you're okay with keeping it as is, let's do that _for now_, and I will open https://github.com/shurcooL/vfsgen/issues/7.

One thought I have is that even if vfsgen had a default variable name that was exported, you may still want to set it explicitly in `appdash-data` because it's a part of its API. If `vfsgen` were to change its generation defaults to have slightly more optimal values (or for any other reason), that would indirectly change the API of `appdash-data`.

That's an argument for not changing `vfsgen` defaults, or alternatively, an argument for keeping the default `vfsgen` VariableName value unexported, so that users must explicitly set it to another value of their choosing if they want it to be exported and hence a part of their API.

> I think so, yes. Having capital letters in filenames is weird.

I agree.

I'll make that change and update everything here as we discussed and let you know to take another look when done. Thanks for review!